src/test/scala/io/iohk/ethereum/network/p2p/FastSyncActorSpec.scala

```diff
@@ -1,278 +0,0 @@
-package io.iohk.ethereum.network.p2p
-
-import akka.actor.ActorSystem
-import akka.testkit.{TestActorRef, TestProbe}
-import akka.util.ByteString
-import io.iohk.ethereum.crypto
-import io.iohk.ethereum.db.dataSource.EphemDataSource
-import io.iohk.ethereum.db.storage._
-import io.iohk.ethereum.domain.BlockHeader
-import io.iohk.ethereum.network.FastSyncActor.{FastSyncDone, SyncFailure}
-import io.iohk.ethereum.network.PeerActor.MessageReceived
-import io.iohk.ethereum.network.p2p.messages.PV62._
-import io.iohk.ethereum.network.p2p.messages.PV63._
-import io.iohk.ethereum.network.{FastSyncActor, PeerActor}
-import org.scalatest.{FlatSpec, Matchers}
-import org.spongycastle.util.encoders.Hex
-
-class FastSyncActorSpec extends FlatSpec with Matchers {
```

Comment:
should we move them sowhere, or implement at least test that checks if fast sync will stop?